examples/local/run.sh

```diff
@@ -1 +1,2 @@
-train --config config.yaml --output output
+# train --config config.yaml --output output --validation
```

Comment:
I think we do not need to change this file.

Reply:
was commited by mistake, removed